openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -124,29 +122,32 @@ OpenAssessment.BaseView.prototype = {
 
         // If we don't have anywhere to put the message, just log it to the console
         if (container === null) {
-            if (msg !== null) { console.log(msg); }
+            if (message !== null) { console.log(message); }
         }
 
         else {
             // Insert the error message
-            var msgHtml = (msg === null) ? "" : msg;
+            var msgHtml = (message === null) ? "" : message;
             $(container + " .message__content", element).html('<p>' + msgHtml + '</p>');
```

Comment:
@andy-armstrong Is msgHtml really html or is it plain text?  I imagine it needs to be escaped and should not be named msgHtml, but not sure.